api/nakadi-event-bus-api.yaml

```diff
@@ -743,6 +743,54 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /subscriptions:
+    post:
```

Comment:
I really think POST is the wrong verb for this endpoint. Since it should indempotent, we should use PUT to create this resource.